include/alpaka/kernel/TaskKernelGpuUniformCudaHipRt.hpp

```diff
@@ -338,11 +359,14 @@ namespace alpaka
                     std::cout << __func__
                         << " BlockSharedMemDynSizeBytes: " << blockSharedMemDynSizeBytes << " B" << std::endl;
 #endif
+                    auto KernelName = kernel::uniform_cuda_hip::detail::uniformCudaHipKernel<TAcc, TDim, TIdx, TKernelFnObj, std::decay_t<TArgs>...>;
```

Comment:
It is not really a type but a function pointer so the upper-case-ness and the name confused me somehow.

Reply:
You are right I will rename it.